### ISSUE TITLE
F2. describe the URL for a webpage associated with a person

### ISSUE THREAD

**Issue by congruili** [USGCRP/gcis-ontology#58]
To describe the URL for a webpage associated with a person, should we use gcis:hasURL or foaf:page?  Note that although we refer mostly to pages on organizational servers, we sometimes refer to personal webpages or to information about people contained on a webpage which isn’t devoted solely to that person (e.g. devoted to other staff, goals, etc.).  See http://xmlns.com/foaf/spec/#term_page  .  Also, we are interested in listing the URL for the page, not the page itself.

**Comment by congruili** [comment]
I think what we need here is a combination of these two properties: [`foaf:page`](http://xmlns.com/foaf/spec/#term_page) and gcis:hasURL. Please comment.

**Comment by zednis** [comment]
We should be able to just use `foaf:page`.

current RDF:

```
<http://data.globalchange.gov/person/1958>
   dcterms:identifier "1958";
   foaf:givenName "Daniel"^^xsd:string;
   foaf:lastName "Abbasi"^^xsd:string;
   gcis:hasURL "http://www.flowcastings.com/Abbasi.php"^^xsd:anyURI;
```

proposed change:

```
<http://data.globalchange.gov/person/1958>
   dcterms:identifier "1958";
   foaf:givenName "Daniel"^^xsd:string;
   foaf:lastName "Abbasi"^^xsd:string;
   foaf:page <http://www.flowcastings.com/Abbasi.php> ;
```

**Comment by justgo129** [comment]
I'm fine with the approach of @zednis.

**Comment by bduggan** [comment]
:+1:

**Comment by justgo129** [comment]
One more thing: @zednis please confirm that foaf:givenName could be also be used for people who use their middle initial as their common name, e.g. as used at:
http://data.globalchange.gov/person/903.thtml

"

**Comment by justgo129** [comment]
Barring opinions to the contrary, I'd say go ahead and prepare the pull request.

**Comment by justgo129** [comment]
Closed #58 due to merged https://github.com/USGCRP/gcis/pull/196.
